variables.tf

```diff
@@ -27,20 +27,44 @@ variable "lacework_image" {
   default     = "lacework/datacollector"
 }
 
+variable "lacework_image_pull_policy" {
+  type        = string
+  description = "The pull policy to use for deploying the Lacework datacollector"
+  default     = "IfNotPresent"
```

Comment:
Shouldn't this be set to `Always`?

Reply:
IIRC, the NIST or CIS are recommending we set the image pull policy to Always. A tool like Checkov does that check: https://github.com/bridgecrewio/checkov/blob/master/checkov/kubernetes/checks/ImagePullPolicyAlways.py

Reply:
I went with the default configuration option on this (https://kubernetes.io/docs/concepts/containers/images/#updating-images) but if you feel like we need to change it to protect against the "tyranny of the default" we can certainly do that.